### PR TITLE
(tlg4102.tlg008)_tlg4102.tlg021

### PR DESCRIPTION
**Pull request by AlisonBabeu**
More cts_.xml updates and adding in page level links.

### CODE REVIEW

File: data/tlg4102/tlg008/tlg4102.tlg008.opp-grc1.xml [hunk 1]
```diff
@@ -73,11 +73,11 @@
               <imprint>
                 <publisher>Oxford University Press</publisher>
                 <pubPlace>Oxford</pubPlace>
-                <date>1844</date>
+                <date>1838</date>
```
Comment (lcerrato) [inline data/tlg4102/tlg008/tlg4102.tlg008.opp-grc1.xml:76]: The date in the cts file is different. (1841 vs 1838)

File: data/tlg4102/tlg013/tlg4102.tlg013.opp-grc1.xml [hunk 2]
```diff
@@ -77,7 +77,7 @@
        </imprint>
```
Comment (lcerrato) [inline data/tlg4102/tlg013/tlg4102.tlg013.opp-grc1.xml:77]: cts file reads 1841

**Review by lcerrato — COMMENTED** [review]
Some of the dates conflict.

File: data/tlg4102/tlg008/tlg4102.tlg008.opp-grc1.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/tlg4102/tlg008/tlg4102.tlg008.opp-grc1.xml:76]: Thanks for catching this, some of the dates are wrong and I was trying to fix that. I will fix that now.

**Review by lcerrato — APPROVED** [review]

**Pull request merged**